client/verta/tests/unit_tests/deployment/test_build.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 
+import pytest
```

Comment:
`pytest` shouldn't be here; standard library imports come first. https://pep8.org/#imports

Reply:
fixed

Reply:
We can just use the PyCharm thing haha